### ISSUE TITLE
Switch default branch from master to main

### ISSUE THREAD

**Issue by DuaneOBrien** [indeedeng/Mariner#33]
We should update this repo to use the main branch rather than master.

**Comment by kevindigo** [comment]
Yes. We seriously considered making this change last week, but ended up deciding to follow the GitHub's advice to hold off until they roll out tools to make the conversion easy. They are targeting a release before the end of 2020, and we plan to use them right away.
